crates/next-core/js/src/entry/error.tsx

```diff
@@ -0,0 +1,4 @@
+import Error, { getStaticProps } from "@vercel/turbopack-next/internal/_error";
+
+export default Error;
+export { getStaticProps };
```

Comment:
this is actually not supported by next.js

https://nextjs.org/docs/advanced-features/custom-error-page#caveats